contributor_docs/es/contributor_guidelines.md

```diff
@@ -0,0 +1,62 @@
+
+### Usando la interfaz de línea de comandos de `git`
```

Comment:
Por favor revisa que todos los títulos coincidan con los de Diana en su PR #6876 

- [Todo acerca de los 'Issues' o problemas](#all-about-issues)
  - [¿Qué son los 'Issues' en GitHub?](#what-are-issues)
  - [Plantillas para reportar 'Issues'](#issue-templates)
    - [Reportar un error](#found-a-bug)
    - [Sugerir una mejora a una funcionalidad existente](#existing-feature-enhancement)
    - [Sugerir una funcionalidad nueva](#new-feature-request)
    - [Abrir una discusión ](#discussion)
- [Trabajando en la base de código de p5.js](#working-on-the-p5js-codebase)
  - [Inicio Rápido para Desarrolladores](#quick-get-started-for-developers)
  - [Utilizando la funcionalidad de edición de Github](#using-the-github-edit-functionality)
  - [Haciendo un fork de p5.js y trabajando desde tu fork](#forking-p5js-and-working-from-your-fork)
    - [Utilizando Github Desktop](#using-github-desktop)
    - [Utilizando la interfaz de línea de comandos de Git](#using-the-git-command-line-interface)
  - [Desglose de la base de código ](#codebase-breakdown)
  - [Configuración de compilación](#build-setup)
  - [Flujo de trabajo de Git](#git-workflow)
    - [Código fuente](#source-code)
    - [Pruebas unitarias](#unit-tests)
    - [Documentación en línea](#inline-documentation)
    - [Internacionalización](https://github.com/processing/p5.js/blob/main/contributor_docs/contributor_guidelines.md#internationalization)
    - [Accesibilidad](#accessibility)
  - [Estándares de código](#code-standard)
  - [Principios de Diseño de Software](#software-design-principles)
- [Pull Requests](#pull-requests)
  - [Creación de un Pull Request](#creating-a-pull-request)
    - [Información sobre el Pull Request](#pull-request-information)
    - [Titulo](#title)
    - [Resuelve ](#resolves)
    - [Cambios](#changes)
    - [Captura de pantalla de los cambios](#screenshots-of-the-change)
    - [Lista de verificación del Pull Request](#pr-checklist)
    - [Rebase y resolución de conflictos](#rebase-and-resolve-conflicts)
  - [Discutir y Corregir](#discuss-and-amend)